services/app-web/src/components/ChangeHistory/ChangeHistoryV2.tsx

```diff
@@ -112,7 +113,10 @@ export const ChangeHistory = ({
                 content: isInitialSubmission ? (
                     <div data-testid={`change-history-record`}>
                         <span className={styles.tag}>Submitted by:</span>
-                        <span> {r.updatedBy}&nbsp;</span>
+                        <span>
+                            {' '}
```

Comment:
nite: use  intentional`&nbsp;` if we are trying to add space -  an empty string rendered is more fragile and easy to autoformat in or out of the file